### PR TITLE
Code of conduct

### PR DESCRIPTION
**Pull request by bobturneruk**
Was approved by @alicepyne against `master` (https://github.com/AFM-SPM/TopoStats/pull/146) so I will merge here without review.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+the consequences for any action they deem in violation of this Code of Conduct:
+
+### 1. Correction
+
+**Community Impact**: Use of inappropriate language or other behavior deemed
+unprofessional or unwelcome in the community.
+
+**Consequence**: A private, written warning from community leaders, providing
+clarity around the nature of the violation and an explanation of why the
+behavior was inappropriate. A public apology may be requested.
```
Comment (ns-rse) [inline CODE_OF_CONDUCT.md:81]: Would an apology be expected if its requested?

**Review by ns-rse — APPROVED** [review]
One minor question but otherwise all good to go I think, thanks.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (bobturneruk) [reply CODE_OF_CONDUCT.md:81]: This is based on a standard CoC - [the contributor covenant](https://www.contributor-covenant.org/). I'd prefer to leave as is. But I see your point.

Comment (ns-rse) [reply CODE_OF_CONDUCT.md:81]: Sure not a problem at all. Its good to go then :+1: and we've one less PR :tada:

**Pull request merged**